src/repository/default.rs

```diff
@@ -79,12 +80,57 @@ impl Repository for $repo {
     async fn get_all_columns_in_database(
         &self,
         database_id: DatabaseId,
+        table_version_ids: Vec<TableVersionId>,
```

Comment:
Minor: use `Option<Vec<TableVersionId>>` and `None` to denote "get all versions"? It's kinda weird to use the empty list to denote all version IDs (though one could argue using `None` for that is as weird)

Reply:
Fine with me, changing.